examples/split_deck.py

```diff
@@ -0,0 +1,31 @@
+#!/usr/bin/env python
+
+from __future__ import print_function
+import sys
+from sunbeam.deck import Deck
+from sunbeam      import action
+
+def split_deck( file_name ):
+    recovery = ('PARSE_RANDOM_SLASH', action.ignore)
+    deck = Deck.parse( file_name, recovery=recovery )
+
+    sections = [ deck.select[ 'RUNSPEC'  ],
+                 deck.select[ 'GRID'     ],
+                 deck.select[ 'EDIT'     ],
+                 deck.select[ 'PROPS'    ],
+                 deck.select[ 'REGIONS'  ],
+                 deck.select[ 'SOLUTION' ],
+                 deck.select[ 'SUMMARY'  ],
+                 deck.select[ 'SCHEDULE' ] ]
+
+    sections = [ v for v in sections if v ]
+    sections.sort( key=lambda cursor: cursor.index )
+
+    for fst, snd in zip( sections, sections[1:] ):
+        with open( fst.value[0] + ".DATA", "w" ) as f:
+            print( Deck( deck[ fst.index : snd.index ] ), file=f )
+
+if __name__ == '__main__':
+    file_name = sys.argv[1] if len( sys.argv ) == 2 else \
+        '../../opm-data/norne/NORNE_ATW2013.DATA'
```

Comment:
This path _only works for you_.  Please just:
```python
if __name__ == '__main__':
    from sys import argv
    if len(argv) != 2:
        exit('Usage: split_deck data.DATA')
    split_deck(argv[1])
```

Reply:
Alright, the assumption of having opm-data was inspired by the other examples. Maybe those should be changed as well?

Reply:
> Maybe those should be changed as well?

YES!!!

Reply:
I agree with Joakim.  The entire idea behind this comes from a disease called _sibling search_.

We would do well to rid us of all remnants of it lest it cripples us more than it already has.

Reply:
I'll change the others as well